libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.cpp

```diff
@@ -175,23 +175,19 @@ void WiFiClientSecure::setBufferSizes(int recv, int xmit) {
   _iobuf_out_size = xmit;
 }
 
-void WiFiClientSecure::stop() {
-  flush();
-  if (_client) {
-    _client->wait_until_sent();
-    _client->abort();
-  }
-  WiFiClient::stop();
+bool WiFiClientSecure::stop(unsigned int maxWaitMs) {
+  bool ret = WiFiClient::stop(maxWaitMs); // calls our virtual flush()
```

Comment:
I see `::stop` calls `::flush` so we're OK there.  Why I did `_client->abort()` as well, I couldn't tell you.  LGTM.